README.md

```diff
@@ -1,15 +1,105 @@
-# Klarna In-App SDK for Klarna Payments - Android Example App
+# Klarna In-App SDK - Android Demo App
+![Platforms](https://img.shields.io/badge/platform-android-lightgrey)
+[![GitHub](https://img.shields.io/github/license/klarna/klarna-mobile-sdk)](https://github.com/klarna/klarna-mobile-sdk/blob/master/LICENSE)
```

Comment:
This one is pointing to the License for the SDK not the demo app repo itself

Reply:
@adrianpalmquist done.